stacker/blueprints/base.py

```diff
@@ -462,6 +463,16 @@ def render_template(self):
         version = hashlib.md5(rendered).hexdigest()[:8]
         return (version, rendered)
 
+    def to_json(self):
+        """Render the blueprint and return the template in json form."""
+
+        variables = []
+        for k in self.get_parameter_definitions():
+            variables.append(Variable(k, 'unused_value'))
```

Comment:
Maybe we could do something a little more intelligent here to cover the most basic cases - the basic python types are probably the easiest.